e2e/assets/imagescan/pre-releases-ignored/fleet.yaml

```diff
@@ -0,0 +1,21 @@
+imageScans:
+# specify the policy to retrieve images, can be semver or alphabetical order
+- policy:
+    # if range is specified, it will take the latest image according to semver order in the range
+    # for more details on how to use semver, see https://github.com/Masterminds/semver
+    # in this test case we're going to use prerelease versions.
+    # as we're specifying * semver will ignore new tags but should not crash the fleet controller
```

Comment:
Having this here is nice for us, the Fleet team, but I think we'd also need an update to our [fleet-docs](https://github.com/rancher/fleet-docs) repo to add info relative to semver and prereleases to [this page](https://fleet.rancher.io/imagescan).

Reply:
sure! I'll add some documentation about pre-releases in there.

Reply:
https://github.com/rancher/fleet-docs/pull/114